test/test_malis_from_pairs.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import malis
+from malis.malis_keras import pairs_to_loss_keras
+from malis.malis_torch import torchloss,pairs_to_loss_torch
+import tensorflow as tf
+import torch
+from torch.autograd import grad
+
+na = np.newaxis
+
+#######################################################
+# TEST 1
```

Comment:
please write a sentence or two what this test is actually doing (what it is actually testing)